storage/s3/src/test/java/io/aiven/kafka/tieredstorage/storage/s3/S3StorageConfigTest.java

```diff
@@ -231,4 +234,28 @@ void withApiCallTimeouts() {
         assertThat(config.apiCallTimeout()).isEqualTo(Duration.ofMillis(5000));
         assertThat(config.apiCallAttemptTimeout()).isEqualTo(Duration.ofMillis(1000));
     }
+
+    @Test
+    void withStorageClass() {
+        final var configs = Map.of(
+                "s3.bucket.name", BUCKET_NAME,
```

Comment:
nit: change indent to 4 spaces
```suggestion
            "s3.bucket.name", BUCKET_NAME,
```
We should have a better tooling to check this automatically.

Reply:
ack

Reply:
done. thanks